Formula/d/ddclient.rb

```diff
@@ -86,7 +80,8 @@ def install
 
   def post_install
     (var/"run").mkpath
-    chmod "go-r", etc/"ddclient.conf"
+
+    chmod "go-r", etc/"ddclient.conf" if (etc/"ddclient.conf").exist?
```

Comment:
```suggestion
    chmod "go-r", pkgetc/"ddclient.conf"
```
We should also remove all the lines below.

Reply:
You can probably just apply this after the bottle commit since it's in `post_install`.

Reply:
The new config path is `etc` and old is `pkgetc`, then I think we should remain `etc` not `pkgetc`, am I right?

Reply:
They changed it back to `pkgetc`: https://github.com/ddclient/ddclient/pull/789

Reply:
should be good now, also update the ddclient.conf file location refs.

Reply:
Please don't force-push once the bottle commit has been created; that destroys status check information.